helpers/shortcuts.ts

```diff
@@ -60,10 +60,16 @@ function transformKeys(keys: string[]) {
     if (Object.keys(keysMap).includes(key)) {
       return keysMap[key];
     }
-    return key.toUpperCase();
+    return toUpperCase(key);
   });
 }
 
+const toUpperCase = (key: string): string => {
+  const char = key.charCodeAt(0);
```

Comment:
:bug: **Bug**
The `toUpperCase` function is incorrectly converting a character to uppercase. The logic `char - 40` is incorrect for converting a lowercase letter to its uppercase equivalent. The correct approach is to use `char - 32` for ASCII characters, as the difference between lowercase and uppercase letters in ASCII is 32. This will lead to incorrect results for any lowercase letter input.

```suggestion
  const upperNumber = String.fromCharCode(char - 32);
```